docker/prod/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
+
+volumes:
+    postgres_data: {}
+    postgres_backup: {}
+    static_dir: {}
+
+services:
+  application:
+    container_name: prod_andela_socials
+    build:
+      context: ../../
+      dockerfile: docker/prod/Dockerfile
+    links:
+     - database
+    env_file:
+      - ../../.env
```

Comment:
Do you have a documented example of what the `.env` file should contain?

Reply:
@andela-johia Yes we do. @tomipaul will share the env-sample.

Reply:
@andela-sjames @johnchuks I will update the `env-sample` to reflect all expected environment variables.